tools/topology/topology2/include/components/copier.conf

```diff
@@ -185,6 +187,8 @@ Class.Widget."copier" {
 	core_id	0
 	cpc		1647
 	bss_size	280
+	num_sink_pins	1
+	num_source_pins	4
```

Comment:
Is it correct? I thought a copier can connect to multiple outputs, right?

Reply:
a sink consumes data, a source generates data. that sounds consistent with the definitions above. For the multi-stream capture this would allow for 4 streams split out of the copier input.

Reply:
You need to look at the 'boundary' of the widget @bardliao, this is different to the source/sink at the widget level.

Copier.DMIC.14.1 has zero sink pins and 2 source pins

Copier.host.13.1 has one sink pin and no source pins.
Same for copier.host.15.1

Reply:
Thanks @plbossart @aiChaoSONG. I understand now.